[2024-01-04T17:12:38.073+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: db_dg.insert_using_python_data manual__2024-01-04T17:06:51.273139+00:00 [queued]>
[2024-01-04T17:12:38.086+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: db_dg.insert_using_python_data manual__2024-01-04T17:06:51.273139+00:00 [queued]>
[2024-01-04T17:12:38.087+0000] {taskinstance.py:2171} INFO - Starting attempt 4 of 6
[2024-01-04T17:12:38.189+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): insert_using_python_data> on 2024-01-04 17:06:51.273139+00:00
[2024-01-04T17:12:38.193+0000] {standard_task_runner.py:60} INFO - Started process 659 to run task
[2024-01-04T17:12:38.196+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'db_dg', 'insert_using_python_data', 'manual__2024-01-04T17:06:51.273139+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/create_database_dag.py', '--cfg-path', '/tmp/tmpcsbbu3rw']
[2024-01-04T17:12:38.202+0000] {standard_task_runner.py:88} INFO - Job 52: Subtask insert_using_python_data
[2024-01-04T17:12:38.459+0000] {task_command.py:423} INFO - Running <TaskInstance: db_dg.insert_using_python_data manual__2024-01-04T17:06:51.273139+00:00 [running]> on host e2a9126be768
[2024-01-04T17:12:38.804+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='prady1900' AIRFLOW_CTX_DAG_ID='db_dg' AIRFLOW_CTX_TASK_ID='insert_using_python_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-04T17:06:51.273139+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-04T17:06:51.273139+00:00'
[2024-01-04T17:12:38.873+0000] {base.py:83} INFO - Using connection ID 'postgres_connect' for task execution.
[2024-01-04T17:12:39.276+0000] {base.py:83} INFO - Using connection ID 'postgres_connect' for task execution.
[2024-01-04T17:12:39.356+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-04T17:12:39.368+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=db_dg, task_id=insert_using_python_data, execution_date=20240104T170651, start_date=20240104T171238, end_date=20240104T171239
[2024-01-04T17:12:39.478+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-04T17:12:39.497+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
